app/controllers/checkouts_controller.rb

```diff
@@ -0,0 +1,10 @@
+class CheckoutsController < ApplicationController
+  before_action :require_login, :allow_anyone
```

Comment:
@derekprior, `allow_anyone` and `require_admin` serve to check (and test) who is allowed to access what book routes. Users (patrons of the library) can only view books and check them out. Admin have the ability to add, delete, and edit books (`admin/books controller`), and can also enjoy the regular benefits of the library - viewing books and checking them out. Thus `books controller` has `allow_anyone`, while `admin/books controller` has `require_admin`.